src/pages/blog/ce-qui-caractérise-le-top-20%-des-devs.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: ../../layouts/ContentLayout.astro
+---
+
+# Ce qui caractérise le top 20% des devs
+
+## Introduction
+
+Lorsque l'on parle des développeurs, il est fréquent de vouloir comprendre ce qui distingue les meilleurs, le fameux top 20%, du reste des développeurs.
+
+Que font-ils différemment ?
+
+Quels sont leurs traits de caractère, leurs habitudes ou leurs compétences spécifiques qui les propulsent au sommet de leur domaine ?
+
+Je suis Oumar, j’ai rejoins le monde de la programmation en septembre 2022 et c’est là tout un tas de question que je me suis posé (et que je me pose toujours d’ailleurs pour sainement progresser 😉).
```

Comment:
Good! You take the time to introduce yourself (but not too much) and you are giving context!

Reply:
Thanks ! 🙂